package.json

```diff
@@ -50,14 +39,12 @@
     "multimatch": "^4.0.0",
     "rxjs": "^6.5.4",
     "tmp-promise": "^3.0.3",
-    "usb-detection": "votingworks/node-usb-detection",
+    "usb-detection": "^4.14.1",
```

Comment:
What was the motivation behind this change?

Reply:
The fix that we had in our fork is now in a release version: https://github.com/MadLittleMods/node-usb-detection/blob/master/CHANGELOG.md#4141---2022-03-10.